styles.css

```diff
@@ -16,14 +16,13 @@
   background-color: #121212; 
   padding: 0.5em; 
   color: white; 
-  position: absolute; 
-  bottom: 0.5em; 
-  left: 0; 
+  /* bottom: 0.5em; 
+  left: 0;  */
   font-weight: 700; 
   z-index: 2; 
-  width: 100%; 
-  font-size: 1.0rem; 
-  opacity: 0.6;
+  width: 100%;
+  font-size: 0.8rem;
+  opacity: 0.8;
```

Comment:
Makes font size smaller. Also, sets opacity to 0.8 because 0.6 is too low color contrast.